src/store/mutations/mutations-model.ts

```diff
@@ -349,3 +349,8 @@ export const mutateRegistrationBusinessType = (state: StateIF, businessType: Bus
   state.stateModel.registration.businessType = businessType
   if (!state.stateModel.ignoreChanges) mutateHaveChanges(state, true)
 }
+
+export const mutateRegistrationBusinessTypeGP = (state: StateIF, businessTypeGP: boolean) => {
+  state.stateModel.registration.businessTypeGP = businessTypeGP
+  if (!state.stateModel.ignoreChanges) mutateHaveChanges(state, true)
```

Comment:
@Lcyc29 In case you're wondering, this is to set a "have changes" flag (as long as we're not ignoring changes, like during initial load) when the user changes this property. This flag is used to warn users if they try to leave this app with unsaved changes.

Reply:
Thanks for the clarification.